internal/namespaces/baremetal/v1/helpers_test.go

```diff
@@ -4,12 +4,18 @@ import (
 	"github.com/scaleway/scaleway-cli/v2/internal/core"
 )
 
+const id = ".ID }}"
```

Comment:
Seems brittle could you use a fmt.Sprintf for the templating? seems to be more readable